src/urllib3/connection.py

```diff
@@ -216,6 +219,10 @@ def _new_conn(self) -> socket.socket:
                 self, f"Failed to establish a new connection: {e}"
             ) from e
 
+        # Audit hooks are only available in versions >= 3.8
```

Comment:
Please replace "versions >= 3.8" with "Python 3.8+", that way when we drop support for 3.8 it will be easy to grep.